xml/ay_bigfile.xml

```diff
@@ -1029,6 +1029,37 @@ exit 0
       <row>
        <entry>
         <para>
+         <literal>self_update</literal>
+        </para>
+       </entry>
+       <entry>
+        <para>
+         This option allows to enable/disable the self-update.
```

Comment:
Better:

    This option allows to enable (set it to <literal>true</literal>)
    or disable (set it to <literal>false</literal> the self-update.